### DISCUSSION TITLE
Stack size actual limit change or editable config

### DISCUSSION THREAD

**Discussion by SquirmyCandy87** [P3pp3rF1y/SophisticatedBackpacks#1093] [General]
Was planning on storing a massive amount of energy using the battery upgrade and some netherite stack upgrades, but it seems to cap out at 4096x stack multiplier, but adding more stack upgrades doesn't affect backpack contents for the battery upgrade past 4096x. (I haven't checked other container upgrades or item size limits). I am wondering if there's a config that can be edited to increase the actual stack level cap past 4096x

**Comment by P3pp3rF1y** [comment]
energy is only limited by the max int value total which is a hard limit because of the way the values are stored. And honestly I don't see a reason for backpack to hold more than many mods can hold only in big multiblocks.
